content/docs/render-props.md

```diff
@@ -225,7 +225,7 @@ function withMouse(Component) {
     render() {
       return (
         <Mouse render={mouse => (
-          <Component {...this.props} mouse={mouse} />
+          <Component mouse={mouse} {...this.props} />
```

Comment:
Leaving the `{...props}` before the `mouse={mouse}` ensures that passing a `mouse` prop can't override what the render prop is trying to pass to the component.

Reply:
I am a bit confused. I thought that if `mouse={mouse}` stay after `{...props}` contrariwise can override attributes in primordial props

Reply:
that's what we want, the render prop should override whatever is passed in